types/container/hostconfig_windows.go

```diff
@@ -15,9 +15,38 @@ func (n NetworkMode) IsNone() bool {
 	return n == "none"
 }
 
+// IsContainer indicates whether container uses a container network stack.
+// Returns false as windows doesn't support this mode
+func (n NetworkMode) IsContainer() bool {
+	return false
+}
+
+// IsBridge indicates whether container uses the bridge network stack
```

Comment:
From a networking perspective, this is one of the most confusing statements I have ever read. Why would `bridge` and `nat` be the same thing?